From 6389405b56d2221d8a7b80c4bb3bd3b4e1a29bab Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Mon, 22 Mar 2010 09:20:49 +1000
Subject: [PATCH 23/37] configure: Always define XINPUT.

This define was removed in e251c9e75afdc "Remove all traces of #ifdef XINPUT
and the matching bits from the configure.ac". Drivers that support multiple
server versions and still check for ifdef XINPUT now always build without
XINPUT support if they're calling XORG_DRIVER_CHECK_EXT(XINPUT, inputproto).

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
Reviewed-by: Dan Nicholson <dbn.lists@gmail.com>
---
 configure.ac |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git xorg-server-1.7.99.902/configure.ac xorg-server-1.7.99.902/configure.ac
index d379b3a..92c34ab 100644
--- xorg-server-1.7.99.902/configure.ac
+++ xorg-server-1.7.99.902/configure.ac
@@ -1122,6 +1122,7 @@ MIEXT_DAMAGE_LIB='$(top_builddir)/miext/damage/libdamage.la'
 MIEXT_DAMAGE_INC='-I$(top_srcdir)/miext/damage'
 
 # XINPUT extension is integral part of the server
+AC_DEFINE(XINPUT, 1, [Support X Input extension])
 XI_LIB='$(top_builddir)/Xi/libXi.la'
 XI_INC='-I$(top_srcdir)/Xi'
 
-- 
1.7.0.2

